release/_publishing.sh

```diff
@@ -195,23 +195,30 @@ function upload_hotfix {
 		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
 	fi
 
-	ssh root@lrdcom-vm-1 mkdir -p "/www/releases.liferay.com/dxp/hotfix/${_PRODUCT_VERSION}/"
+	ssh root@lrdcom-vm-1 "echo 'Connection successful'" &> /dev/null
 
-	#
-	# shellcheck disable=SC2029
-	#
-
-	if (ssh root@lrdcom-vm-1 ls "/www/releases.liferay.com/dxp/hotfix/${_PRODUCT_VERSION}/" | grep -q "${_HOTFIX_FILE_NAME}")
+	if [ $? -eq 0 ]
```

Comment:
@kiwm, is the result stored in `$?` the result of `ssh` or `echo`? Actually, is the `echo` really necessary?